docker/env/version

```diff
@@ -1 +1 @@
-0.54
+0.55-urllib3
```

Comment:
Let's remove the spare suffix before merge.

The test of this PR might create unofficial image (0.55) on builders, if this pr is changed, or other docker change uses the 0.55 version. The docker image will not match with official version.

Reply:
Yes we are aware of this issue, once a PR is approved we change to a formal version. The issue get ugly when too PRs are doing changes to this version, hence the use of the name